lib/src/main/java/org/team100/lib/commands/drivetrain/DriveWithProfileNote.java

```diff
@@ -64,7 +69,7 @@ public DriveWithProfile(
         Constraints100 thetaContraints = new Constraints100(m_limits.getMaxAngleSpeedRad_S(),
                 m_limits.getMaxAngleAccelRad_S2() / 4);
         Constraints100 driveContraints = new Constraints100(m_limits.getMaxDriveVelocityM_S(),
-                m_limits.getMaxDriveAccelerationM_S2() / 4);
+                m_limits.getMaxDriveAccelerationM_S2() );
```

Comment:
this is very high accel, which will spin the tires and mess up the odometry somewhat.  are you sure you want that?